package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "metrics",
+  "description": "Stores performance metrics for rust-analyzer.",
+  "version": "1.0.0",
+  "devDependencies": {
+    "typescript": "^4.5.2",
+    "@types/plotly.js": "^1.54.16"
+  },
+  "scripts": {
+    "build": "./node_modules/typescript/bin/tsc -p tsconfig.json",
+    "start": "./node_modules/typescript/bin/tsc -p tsconfig.json --watch"
+  },
+  "license": "ISC",
```

Comment:
@matklad we had no license before. I suppose this is as good as any, but we should probably check with the other two contributors?

Reply:
Let's make this `"license": "MIT OR Apache-2.0",` just in an unlikely case we want to move some JS code to/from vscode plugin

@R4DIC4L @homersimpsons are you OK with licensing the code thusly?

Reply:
I hereby agree to license my contributions to this repository under the MIT or Apache-2.0 licenses.

Reply:
I also agree to license my contributions to this repository under the MIT or Apache-2.0 licenses.

Reply:
I'll change the license shortly.

Reply:
@matklad definitely !

I hereby agree to license my contributions to this repository under the MIT or Apache-2.0 licenses.